src/components/PathwaysList/PathwaysList.tsx

```diff
@@ -1,174 +1,109 @@
-import React, { FC, ReactNode, useState } from 'react';
-import { makeStyles } from '@material-ui/core/styles';
-import clsx from 'clsx';
+import React, { FC } from 'react';
+import { Pathway, EvaluatedPathway } from 'pathways-model';
 import { Service } from 'pathways-objects';
-import { Pathway, EvaluatedPathway, CriteriaResult } from 'pathways-model';
-
-import styles from './PathwaysList.module.scss';
-import indexStyles from 'styles/index.module.scss';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import Graph from 'components/Graph';
-import { IconProp } from '@fortawesome/fontawesome-svg-core';
-import { usePathwayContext } from 'components/PathwayProvider';
+import { faPlus, faEdit, faTrashAlt } from '@fortawesome/free-solid-svg-icons';
 import {
-  faPlay,
-  faPlus,
-  faMinus,
-  faChevronUp,
-  faChevronDown,
-  faCaretDown
-} from '@fortawesome/free-solid-svg-icons';
+  Button,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from '@material-ui/core';
 
-const useStyles = makeStyles(
-  theme => ({
-    'pathway-element': {
-      backgroundColor: theme.palette.background.default
-    },
-    title: {
-      color: theme.palette.text.primary
-    }
-  }),
-  { name: 'PathwaysList' }
-);
+import Loading from 'components/elements/Loading/Loading';
 
-interface PathwaysListElementProps {
-  evaluatedPathway: EvaluatedPathway;
-  criteria?: CriteriaResult;
-  callback: Function;
+import styles from './PathwaysList.module.scss';
+
+interface PathwaysTableProps {
+  pathways: EvaluatedPathway[];
+  deleteButton: Function;
+  editButton: Function;
 }
 
+const PathwaysTable: FC<PathwaysTableProps> = ({
+  pathways,
+  deleteButton,
+  editButton,
+}) => {
+  return (
+    <TableContainer className={styles.pathwayList}>
+      <Table aria-label="pathway list">
+        <TableHead>
+          <TableRow>
+            <TableCell>Pathway Name</TableCell>
+            <TableCell>Status</TableCell>
+            <TableCell>Last Updated</TableCell>
+            <TableCell></TableCell>
+          </TableRow>
+        </TableHead>
+
+        <TableBody>
+          {pathways.map((pathway) => (
+            <TableRow key={pathway.pathway.name}>
+              <TableCell component="th" scope="row">
+                {pathway.pathway.name}
+              </TableCell>
+              <TableCell>draft</TableCell>
+              <TableCell>2 days ago</TableCell>
+              <TableCell align="right">
+                <Button
+                  className={styles.editButton}
+                  color="primary"
+                  size="small"
+                  startIcon={<FontAwesomeIcon icon={faEdit} />}
+                >
```

Comment:
Can we have this Edit button bring up the pathway view, the same way that the Select Pathway button previously did? I think it's only a couple line change -- add `onClick={() => editButton(pathway)}` here and down below where the PathwaysTable is rendered, change the editButton to `editButton={(pathway: Pathway): void => callback(pathway)}`

Reply:
Edit button will now show the pathway view